app/lib/frontend/templates/views/security.mustache

```diff
@@ -0,0 +1,18 @@
+{{! Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
+    for details. All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file. }}
+<div style="max-width: 600px">
+  <h2>Security</h2>
+  <p>
+      The Dart team takes the security of Pub and its published packages seriously.
+      This page describes how to report any vulnerabilities you may find.
+  </p>
+
+  <h3>Reporting vulnerabilities</h3>
+  <p>
+      In the rare event that you find a vulnerability in Pub site itself, email us at
+      <a href="mailto:product-security@google.com">product-security@google.com</a>.
+      For more information about how Google handles security issues, see
+      <a href="https://www.google.com/about/appsecurity/">Google’s security philosophy</a>.
```

Comment:
@mit-mit, do you have any comments on the contents here?

If we want `product-security@google.com` handling, we probably have to do some setup.

Reply:
I already did that (see cl/261665758)